db/migrate/20231109011155_create_ai_personas.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+#
+class CreateAiPersonas < ActiveRecord::Migration[7.0]
+  def change
+    create_table :ai_personas do |t|
```

Comment:
I think since admins will be creating these we should have a `created_by_id` linking to a User. Maybe even have a `status` or `enabled` bool, so people can turn them on/off easily without deleting them?

Reply:
agree totally ... enabled and created_by_id are very handy

Reply:
going to do this in a followup PR